sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -76,50 +75,69 @@ export interface ChatClientOptions extends PipelineOptions {
 }
 
 // @public
-export interface ChatMessage extends Omit<RestChatMessage, "senderId" | "content"> {
+export interface ChatMessage {
     content?: ChatMessageContent;
-    sender?: CommunicationUserIdentifier;
+    createdOn: Date;
+    deletedOn?: Date;
+    editedOn?: Date;
+    id: string;
+    senderCommunicationIdentifier?: CommunicationIdentifierModel;
+    senderDisplayName?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
+// @public
+export interface ChatMessageContent {
+    initiatorCommunicationIdentifier?: CommunicationIdentifierModel;
```

Comment:
```suggestion
    initiator?: CommunicationIdentifier;
```

Reply:
What does `initiator` mean for a `ChatMessageContent`?  Is that the initiator of the thread? I wouldn't expect this on the message.

Reply:
@juancamilor  @amrelroumy-msft

Reply:
`initiator` is for system messages like participant added/removed, it would represent the participant that initiated that event